easybuild/easyconfigs/i/Infomap/Infomap-20190308-GCC-8.2.0-2.31.1.eb

```diff
@@ -0,0 +1,27 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+easyblock = 'MakeCp'
+
+name = 'Infomap'
+version = '20190308'
+local_commit = 'bd65c71'
+
+homepage = 'https://www.mapequation.org/code.html#Linux'
+description = """Multi-level network clustering based on the Map equation."""
+
+toolchain = {'name': 'GCC', 'version': '8.2.0-2.31.1'}
+toolchainopts = {'pic': True}
+
+files_to_copy = ['%(name)s']
+source_urls = [GITHUB_LOWER_SOURCE]
+sources = [{'download_filename': '%s.tar.gz' % local_commit, 'filename': SOURCE_TAR_GZ}]
+checksums = ['4f4dcc718faf3b1bac06273a5636e1fb8e250b87f442085cd6b3a0ae0ce0418e']
+
+github_account = 'mapequation'
```

Comment:
this belongs above `source_urls` (but this got probably mixed up due to `--inject-checksums`?)